Functions.Templates/Templates/IoTHubServiceBusQueueTrigger-JavaScript/metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+  "defaultFunctionName": "IoTHubServiceBusQueueTriggerJS",
```

Comment:
Same here function apps allow only one language per function app. No need to have "JS" appended to the name anymore.